ui/react-components/components/AppointmentListSidePanelTree/AppointmentListSidePanelTree.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import CheckboxTree from "react-checkbox-tree";
+import "react-checkbox-tree/lib/react-checkbox-tree.css";
+import {
+  MdCheckBox,
+  MdCheckBoxOutlineBlank,
+  MdAddBox,
+  MdIndeterminateCheckBox,
+  MdAdd,
+  MdRemove
+} from "react-icons/md";
+import "./AppointmentListSidePanelTree.module.scss";
+import { injectIntl } from "react-intl";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+
+const AppointmentListSidePanelTree = props => {
```

Comment:
Why is this called AppointmentListSidePanelTree?

Reply:
Renamed it as AppSpecialityFilter

Reply:
@bsneha90 @binduak, I will be merging this PR. As of now we are going ahead with the `react-checkbox-tree` for the tree.